secure.go

```diff
@@ -56,6 +57,8 @@ type Options struct {
 	ContentSecurityPolicy string
 	// PublicKey implements HPKP to prevent MITM attacks with forged certificates. Default is "".
 	PublicKey string
+	// Referrer Policy allows sites to control when browsers will pass the Referer header to other sites. Default is "".
```

Comment:
My understanding is the "Referrer-Policy" HTTP header (with two 'r's) governs which information should be included in the "Referer" HTTP header (with a single 'r').
The "Referer" header is spelt with a single 'r' due to a misspelling in the original RFC, see the Wikipedia page: https://en.wikipedia.org/wiki/HTTP_referer.

I believe that my comment is correct here, referencing the two spellings for the two headers, but I could be mistaken and just can't see it.